asab/web/auth/authorization.py

```diff
@@ -189,16 +190,28 @@ def require_tenant_access(self):
 
 	def authorized_resources(self) -> typing.Optional[typing.Set[str]]:
 		"""
-		Return the set of authorized resources.
+		DEPRECATED. Return the set of authorized resources.
```

Comment:
If this is deprecated method, you can put that information into LOG_OBSOLETE perhaps...

Reply:
on second thought i'm making the method private instead. LOG_OBSOLETE is called one level higher, in the method that uses this one.